src/main/java/com/example/rets_api/repository/PropertyRepositoryQuerydsl.java

```diff
@@ -60,304 +76,325 @@ public List<PropertyEntity> fetchAll(PropertyFilter filterParams) {
             query = query.where(viewData.cityLightIndicator.eq(filterParams.getViewDataCityLightIndicator()));
 
-        if (!filterParams.getViewDataMountainIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+        if (nonDefaultValue(filterParams.getViewDataMountainIndicator()))
             query = query.where(viewData.mountainIndicator.eq(filterParams.getViewDataMountainIndicator()));
 
-        if (!filterParams.getViewDataRiverIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+        if (nonDefaultValue(filterParams.getViewDataRiverIndicator()))
             query = query.where(viewData.riverIndicator.eq(filterParams.getViewDataRiverIndicator()));
 
-        if (!filterParams.getViewDataLakeIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+        if (nonDefaultValue(filterParams.getViewDataLakeIndicator()))
             query = query.where(viewData.lakeIndicator.eq(filterParams.getViewDataLakeIndicator()));
 
-        if (!filterParams.getViewDataGolfCourseIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+        if (nonDefaultValue(filterParams.getViewDataGolfCourseIndicator()))
             query = query.where(viewData.golfCourseIndicator.eq(filterParams.getViewDataGolfCourseIndicator()));
 
-        if (!filterParams.getViewDataWaterIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+        if (nonDefaultValue(filterParams.getViewDataWaterIndicator()))
             query = query.where(viewData.waterIndicator.eq(filterParams.getViewDataWaterIndicator()));
+    }
 
-        if (!filterParams.getFinancialDataTradeIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+    private void addFinancialDataFiltersToQuery() {
+        if (nonDefaultValue(filterParams.getFinancialDataTradeIndicator()))
             query = query.where(financialData.tradeIndicator.eq(filterParams.getFinancialDataTradeIndicator()));
 
-        if (!filterParams.getFinancialDataLeaseIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+        if (nonDefaultValue(filterParams.getFinancialDataLeaseIndicator()))
             query = query.where(financialData.leaseIndicator.eq(filterParams.getFinancialDataLeaseIndicator()));
+    }
 
-        if(!filterParams.getFinancialDataRentalAmountStart().equals(DEFAULT_LONG_VALUE) || !filterParams.getFinancialDataRentalAmountEnd().equals(DEFAULT_LONG_VALUE))
-            query = configureRentAmountInTheQuery(query);
-
-        if (!filterParams.getAnimalPolicyAnimalPermitted().equals(Indicator.DEFAULT_ENUM_VALUE))
+    private void addAnimalPolicyFiltersToQuery() {
+        if (nonDefaultValue(filterParams.getAnimalPolicyAnimalPermitted()))
             query = query.where(animalPolicy.animalsPermitted.eq(filterParams.getAnimalPolicyAnimalPermitted()));
+    }
 
-        if (!filterParams.getLotDataGolfCourseLotIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+    private void addLotDataFiltersToQuery() {
+        if (nonDefaultValue(filterParams.getLotDataGolfCourseLotIndicator()))
```

Comment:
@taisemarques   This comment is just a suggestion.
Since we are calling filterParams.method() two times for each if statement, would it be better to create variables to store the result of the method call, like that we're gonna drop our number of method call by half.
The only inconvenient would be the storage maybe.
Let me know what do you think. Thanks!